charts/rancher-thanos/v0.0.2/values.yaml

```diff
@@ -21,8 +21,8 @@ grafana:
     tag: 6.3.6
     pullPolicy: IfNotPresent
     proxy:
-      repository: rancher/nginx
-      tag: 1.17.4-alpine
+      repository: cnrancher/nginx
+      tag: stable
```

Comment:
Use a specific version considering that we might need to update it in follow-up releases.

Reply:
fixed